### PR TITLE
add-apt-repository, apt-cache: add Indonesian translation

### PR DESCRIPTION
**Pull request by punkestu**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution, seems like you used the existing branch from #12356 and the `add-apt-repository.md` file is present here too. You could either delete the file from this PR or you could close the other pull request (anything would work).

Btw, for `apt-cache.md`, can you translate the placeholders inside `{{}}` too?

File: pages.id/linux/apt-cache.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# apt-cache
+
+> Alat query paket untuk Debian dan Ubuntu.
```
Comment (reinhart1010) [inline pages.id/linux/apt-cache.md:3]:
```suggestion
> Pencari paket untuk Debian dan Ubuntu.
```

Might be better to simplify this `query tool` term.

File: pages.id/linux/add-apt-repository.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# add-apt-repository
+
+> Mengelola definisi repositori `apt`.
```
Comment (reinhart1010) [inline pages.id/linux/add-apt-repository.md:3]:
```suggestion
> Kelola definisi repositori `apt`.
```

Can you please change this (and subsequent) sentences to use imperative mood (see [this](https://share.reinhart1010.id/tldrguidelinesidspecific) and [this](https://share.reinhart1010.id/tldridimperativemood))?

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Thanks for contributing! Here are some of my notes to review.

**Review by punkestu — COMMENTED** [review]
I have fixed it

**Review by kbdharun — APPROVED** [review]

File: pages.id/linux/add-apt-repository.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# add-apt-repository
+
+> Kelola definisi repositori `apt`.
+> Informasi lebih lanjut: <https://manned.org/apt-add-repository>.
+
+- Menambah repositori `apt` baru:
```
Comment (reinhart1010) [inline pages.id/linux/add-apt-repository.md:6]:
```suggestion
- Tambah repositori `apt` baru:
```

File: pages.id/linux/add-apt-repository.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`add-apt-repository {{repositori}}`
+
+- Menghilangkan sebuah repositori `apt`:
```
Comment (reinhart1010) [inline pages.id/linux/add-apt-repository.md:10]:
```suggestion
- Hilangkan sebuah repositori `apt`:
```

File: pages.id/linux/add-apt-repository.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`add-apt-repository --remove {{repositori}}`
+
+- Memperbarui cache paket setelah menambahkan sebuah repositori:
```
Comment (reinhart1010) [inline pages.id/linux/add-apt-repository.md:14]:
```suggestion
- Perbarui cache paket setelah menambahkan sebuah repositori:
```

File: pages.id/linux/add-apt-repository.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+`add-apt-repository --update {{repositori}}`
+
+- Mengizinkan sumber paket untuk didownload dari repositori:
```
Comment (reinhart1010) [inline pages.id/linux/add-apt-repository.md:18]:
```suggestion
- Izinkan sumber paket untuk diunduh dari repositori:
```

File: pages.id/linux/apt-cache.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
+# apt-cache
+
+> Pencari paket untuk Debian dan Ubuntu.
+> Informasi lebih lanjut: <https://manpages.debian.org/latest/apt/apt-cache.8.html>.
+
+- Mencari paket di sumber yang sudah kamu miliki:
```
Comment (reinhart1010) [inline pages.id/linux/apt-cache.md:6]:
```suggestion
- Cari paket di sumber yang sudah dimiliki:
```

Avoid using first-person nouns such as `kamu`.

File: pages.id/linux/apt-cache.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+`apt-cache search {{query}}`
+
+- Menampilkan informasi tentang sebuah paket:
```
Comment (reinhart1010) [inline pages.id/linux/apt-cache.md:10]:
```suggestion
- Tampilkan informasi tentang sebuah paket:
```

File: pages.id/linux/apt-cache.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+`apt-cache show {{paket}}`
+
+- Menampilkan apakah sebuah paket sudah terinstal dan paling terbaru:
```
Comment (reinhart1010) [inline pages.id/linux/apt-cache.md:14]:
```suggestion
- Tampilkan apakah sebuah paket sudah terinstal dan paling terbaru:
```

File: pages.id/linux/apt-cache.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+
+`apt-cache policy {{paket}}`
+
+- Menampilkan dependensi sebuah paket:
```
Comment (reinhart1010) [inline pages.id/linux/apt-cache.md:18]:
```suggestion
- Tampilkan dependensi sebuah paket:
```

File: pages.id/linux/apt-cache.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+
+`apt-cache depends {{paket}}`
+
+- Menampilkan paket yang bergantung pada paket tertentu:
```
Comment (reinhart1010) [inline pages.id/linux/apt-cache.md:22]:
```suggestion
- Tampilkan paket yang bergantung pada paket tertentu:
```

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Hi, can you also apply these changes? Thank you!

**Review by reinhart1010 — APPROVED** [review]
Thanks!

**Pull request merged**